src/content/docs/blog/2025/06/30/mid-year-update/index.md

```diff
@@ -0,0 +1,185 @@
+
+Over the last few months, things may have seemed unusually quiet, however rest assured that there has been A LOT going on in the background. As such, we are preparing a short series of blog posts to go over the relevant topics in the coming weeks.
+
+For this blog post, we are going to cover our infrastructure port, along with the process of rebuilding our entire package repository.
+
+**The TL;DR is that:**
+
+- All core AerynOS tooling is now written in Rust
+- Every recipe in the repository has been rebuilt (twice!) with many packages then having been updated to newer versions after the rebuilds were completed
+- A CDN has been implemented for faster package installation and ISO downloads
+
+## New Infrastructure tooling
+
+When delivering a Linux distribution, its infrastructure and associated processes effectively act as the "spine" of the project. But spine surgery can be a delicate affair, particularly when it comes to rehabilitation after successful surgery.
+
+For us, this cycle has been particularly demanding, as we have completed an MVP port of our infrastructure tooling code to Rust, meaning that all core AerynOS tooling has now fully transitioned away from DLang.
+
+We have covered the reasons for transitioning away from DLang [previously](https://aerynos.com/blog/2023/09/06/oxidised-moss/), and it's fair to say that we are already feeling the benefits of easy and native reuse of code in our tooling repositories and welcoming more Rust contributors into our community.
+
+### Why now?
+
+Earlier this year, our existing DLang build infrastructure started showing signs of instability and required more and more manual intervention to successfully land packages. 
+
+Given our prior decision to transition our tooling over to Rust, we had already stopped further development of the DLang based infra. Hence, we decided to accelerate our transition timeline for the infra re-write to Rust with tarkah and ermo leading the development activity, which began at the end of March.
+
+Towards the end of May, we put the first prototype infra to the test, and then iteratively fixed bugs and built out missing functionality to the point of being able to put our Minimum Viable Product (MVP) into production on our build infrastructure.
+
+This MVP will serve as the development base of the code that will be used for all future package builds.
+
+<details>
+<summary>What do we mean by infra?</summary>
+
+Our [infra](https://github.com/AerynOS/infra) is comprised of the Summit, Avalanche and Vessel service components:
+
+- **Summit:** Package build-controller, build-orchestrator and build-dashboard, monitors recipes tree and automatically builds new, incoming recipes once they show up
+- **Avalanche:** build agent middleware, takes build orders from summit and builds them with boulder on a remote system, sends build logs back in real time to summit, and reports the build result to summit at the end of the build.
```

Comment:
```suggestion
- **Avalanche:** Build agent middleware, takes build orders from Summit and builds them with boulder on a remote system, sends build logs back in real time to Summit, and reports the build result to summit at the end of the build.
```

Since the other two sections start with a capital letter as well, and the name of the infra parts are also capitalised elsewhere. Not sure about "boulder"